src/sidebar/util/type-coercions.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Type conversion methods that gently coerce values that other parts
```

Comment:
Given that these functions seem quite generic, I think we should be more specific that they are for coercing incoming configuration values to an expected type or format.